### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by carolinamenezes**
#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,126 @@
...
+To locally test your plugins as you code, you'll need to use the `yarn watch` command, which is responsible for creating a [symlink](https://en.wikipedia.org/wiki/Symbolic_link) named `vtex-test`.
+
+To use the `yarn watch` command, you must add `$HOME/.vtex/dev/bin` to your `PATH` environment variable. Take the following steps:
+
+>⚠️ ***Keep in mind:** these steps may vary depending on your operating system.*
+
+1. Open the `~/.bashrc` file (or the corresponding shell configuration file).
+2. At the end of this file, add the line: `export PATH=$PATH:$HOME/.vtex/dev/bin`.
+3. Save it.
+4. Run the command source `~/.bashrc` (or corresponding file).
```
Comment (VerasThiago) [inline README.md:38]: The word `source` should be in the same syntax as `~/.bashrc`.

source `~/.bashrc` => `source ~/.bashrc`

File: README.md [hunk 2]
```diff
@@ -1,42 +1,126 @@
...
+Develop your plugin commands inside the `/src/commands/` folder, referring to the [oclif API reference](https://oclif.io/docs/commands) and `/src/commands/hello.ts` example file.
+
+Also, if necessary, feel free to create libraries, clients, and [modules](https://oclif.io/docs/running_programmatically#sharing-code-with-modules) inside your `src` folder.
+
+>ℹ️ *Please refer to our [Toolbelt design guidelines](https://github.com/vtex/ux-writing/blob/new-content/docs/text-patterns/toolbelt-cli.md) for keeping up with the **best practices** when developing your plugin.*
+
+### Step 3: Testing your plugin
+
+1. Run `yarn link` in your plugin project.
+2. Now, in the **cloned** Toolbelt directory, run `vtex link @vtex/cli-plugin-{name}`.
```
Comment (VerasThiago) [inline README.md:66]: `vtex link @vtex/cli-plugin-{name}` => `yarn link @vtex/cli-plugin-{name}`

File: README.md [hunk 3]
```diff
@@ -1,42 +1,126 @@
...
+3. Run `yarn watch` in both the cloned `vtex/toolbelt` and in the plugin directory.
```
Comment (VerasThiago) [inline README.md:67]: How about use the code syntax to "plugin directory" ? 

plugin directory => `plugin directory`

Comment (VerasThiago) [reply README.md:67]: Also change "directory" to "repo" or "repository"

`plugin directory` => `plugin repo`

File: README.md [hunk 4]
```diff
@@ -1,42 +1,126 @@
...
+3. Run `yarn watch` in both the cloned `vtex/toolbelt` and in the plugin directory. 
+
+A symlink named `vtex-test` will be created and you'll be able to locally test your plugin as you develop it. That means, for example, that you can test your plugin commands on any VTEX IO app by locally running `vtex-test {command-name}`.
+
+### Step 4: Standardizing and deploying your plugin
+
+If you're satisfied with the tests performed and the developed plugin:
+
+1. Access your profile on [npm](https://www.npmjs.com/). As a part of the `vtex` organization, you'll have access to [generate a new **publish** token.](https://docs.npmjs.com/creating-and-viewing-access-tokens)
+2. Save the generated token.
```
Comment (VerasThiago) [inline README.md:76]: How about  "Copy the generated token" instead of "save"?

File: README.md [hunk 5]
```diff
@@ -1,42 +1,126 @@
...
+
+### [OPTIONAL] Step 5: Adding your new plugin to the Toolbelt
+
+The [Toolbelt](https://github.com/vtex/toolbelt) project adheres to a strict pull request review workflow.
+
+To request including your plugin in the Toolbelt, update the [Toolbelt `dependencies`](https://github.com/vtex/toolbelt/blob/master/package.json#L52) and the [`CHANGELOG.md`](https://github.com/vtex/toolbelt/blob/master/CHANGELOG.md) file. Open a pull request, explaining what your plugin does, wait for comments from reviewers, discuss and fix any related issues. Once you receive an LGTM, you'll be able to merge it to the master branch.
+
+>⚠️ ***Keep in mind:** If your plugin is not accepted in the [Toolbelt](https://github.com/vtex/toolbelt) project, you can still use it **locally**. For that, you must run `yarn add {path}` in the **cloned** Toolbelt folder, where `{path}` must be replaced with the [npm](https://www.npmjs.com/) or relative path of your plugin. In these cases, however, VTEX does not support, neither takes any responsibility for any damages caused by the developed plugin.*
```
Comment (VerasThiago) [inline README.md:103]: I think this part was no clear and correct enough.

1 - To run a custom plugin locally you need to go inside de cloned `toolbelt` repo and fire the command `yarn add PLUGIN_NAME`, then run `yarn watch` and use `vtex-test` to use it.

2 - Fortunately the new `toolbelt` major (3.x) will contain a new command called `plugins` that will enable a user to install plugins directly inside `toolbelt`, for exemple: `vtex plugins install lighthouse` that will find for `@vtex/cli-plugin-lighthouse` on `npm`. For third parties plugins you can use `vtex plugins install PLUGIN_NAME` that can be in the format `@ORG/PLUGIN_NAME`

File: README.md (hunk 1 above)
Comment (carolinamenezes) [inline README.md:38]:
```suggestion
4. Run the command `source ~/.bashrc` (or corresponding file).
```

**Review by VerasThiago — APPROVED** [review]
LGTM

**Pull request merged**